recipes/responder/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge dev
```

Comment:
after some searching, the syntax is probably different, maybe try something like:

```yaml
channel_sources:
  - conda-forge/label/dev,defaults
```

or 

```yaml
channel_sources:
  - conda-forge/label/dev,defaults
```